docs/api-docs/analytics/BODL.md

```diff
@@ -0,0 +1,432 @@
+}
+</script>
+<!-- Sample Pixel Code End: Add to Wishlist -->
+```
+### Order complete
+This snippet pulls down a series of objects from a completed order, splitting them into nested arrays for physical items, digital items, and gift certificates.
+
+```javascript
+<!-- Sample Pixel Code Start: Purchase Event -->
+<script>
+fetch('/api/storefront/order/{{checkout.order.id}}', {
+  credentials: 'same-origin'
+})
+.then(function(response) {
+  return response.json();
+})
+.then(function(orderJson) {
+  var orderQty = 0;
+	var lineItems = [];
+
+	for (i = 0; i < orderJson.lineItems.physicalItems.length; i++) {
+    var thisItem = orderJson.lineItems.physicalItems[i];
+    orderQty += thisItem.quantity;
+    lineItems.push({
+      "content_id": thisItem.productId,
+      "content_name": thisItem.name,
+      "currency": orderJson.currency.code,
+      "price": thisItem.salePrice,
+      "value": thisItem.extendedSalePrice,
+      "quantity": thisItem.quantity,
+      "content_type": "product_group"
+    });
+	}
+
+	for (i = 0; i < orderJson.lineItems.digitalItems.length; i++) {
+    var thisItem = orderJson.lineItems.digitalItems[i];
+    orderQty += thisItem.quantity;
+    lineItems.push({
+      "content_id": thisItem.productId,
+      "content_name": thisItem.name,
+      "currency": orderJson.currency.code,
+      "price": thisItem.salePrice,
+      "value": thisItem.extendedSalePrice,
+      "quantity": thisItem.quantity,
+      "content_type": "product_group"
+    });
+	}
+
+	for (i = 0; i < orderJson.lineItems.giftCertificates.length; i++) {
+    var thisItem = orderJson.lineItems.giftCertificates[i];
+    orderQty += thisItem.quantity;
+    lineItems.push({
+      "content_id": thisItem.type,
+      "content_name": thisItem.name,
+      "currency": orderJson.currency.code,
+      "price": thisItem.amount,
+      "value": thisItem.amount,
+      "quantity": thisItem.quantity,
+      "content_type": "product_group"
+    });
+	}
+
+  sampleAnalyticsProvider.instance('<%= property_id %>').track('Purchase', {
+      "contents": lineItems,
+      "value": orderJson.orderAmount,
+      "quantity": orderQty,
+      "currency": orderJson.currency.code
+  });
+});
+</script>
+<!-- Sample Pixel Code End: Purchase Event -->
+```
+
+### Registration
+
+This simple snippet tracks account creation.
+
+```javascript
+<!-- Sample Pixel Code Start: Registration -->
+<script>
+if (window.location.pathname.indexOf('/login.php') === 0 && BODL.getQueryParamValue('action') === 'account_created') {
+  sampleAnalyticsProvider.instance('<%= property_id %>').track('Registration');
+}
+</script>
+<!-- Sample Pixel Code End: Registration -->
+```
+
+### Search
+This snippet tracks when an end-user searches for products. Please take note that there is a built-in tracker for the category. However, we have commented out the tracker due to known distortions, which cause reporting of only the first category ID. Only reactivate if needed; use it at your own risk.
```

Comment:
```suggestion
This snippet tracks when an end-user searches for products. Please take note that there is a built-in tracker for the category. However, the tracker is commented out due to known distortions, so only the first category ID is reported. Only reactivate if needed; use it at your own risk.
```

Reply:
Changed to 3rd person to avoid personal ("we") pronoun

Reply:
I made all the changes except for "the tracker is commented out" because this text is passive voice.